### ISSUE TITLE
Mail are not sent to customer neither merchant after upgrade from 1.7.6.1 to 1.7.8.0

### ISSUE THREAD

**Issue by LollipopAdmin** [PrestaShop/PrestaShop#26470]
#### Describe the bug

After upgrade, mail are not sent to customers when they order, neither to merchant.
I tried to send mail from the test email on Advanced settings > Email but it does not work neither.
I'm using standard email setup /usr/sbin/sendmail

#### Expected behavior
Mails should be sent.

#### Steps to Reproduce

1. Go to 'Back-office > Advanced Settings > Email > Test configuration email'
2. Click on 'Send a test mail'

I also do see some errors like that in the Logs of Prestsahop but not clear what they mean, still they are the same as I sent an test email - but logs are not providing more info than that. 
**Swift Error: Expected response code 220 but got an empty response** 

**Screenshots**
![image](https://user-images.githubusercontent.com/75253748/139326171-07c43eaa-585f-477e-9b97-716acb17f2fb.png)

If applicable, add screenshots or screen recordings to help explain your problem.

#### Additional information

* PrestaShop version: 1.7.8.0
* PHP version: 7.3.31 also tried with 7.4 that I applied after upgrade from Prestashop 1.7.6.1 to 1.7.8.0 but does not work any better. 
* 1 Click upgrade version : v4.12.0

**Comment by khouloudbelguith** [comment]
Hi @LollipopAdmin,

I did not manage to reproduce the issue after an upgrade from PS1769 to PS1780 using the last version 1-click upgrade v4.13.0.
I attached a screen record
https://watch.screencastify.com/v/ajvkRDa3viLBBac60P94

Thanks to check and feedback.

**Comment by Stephane1874** [comment]
Hello @LollipopAdmin ,

I had a similar problem on my store corrected with this solution.
https://github.com/PrestaShop/PrestaShop/issues/22262#issuecomment-754548289

The cause: https://github.com/PrestaShop/PrestaShop/pull/20124#issue-654168951

If this is the problem you can also correct it from your hosting space by activating the proc_open function from your php.ini file: https://github.com/PrestaShop/PrestaShop/issues/25380

I hope these links can help you unblock yourself from this situation.

**Comment by LollipopAdmin** [comment]
Hello, 

Thanks to all for looking at my issue.  

I finally fixed my issue by setting up a real SMTP Server from my host provider One.com.
I have used TLS encryption on dedicated Port 587 with SMTP server (mailout.one.com) as defined in one.com documentation.
Since then it is working like a charm...

Concusion:
I'm missing a bit more pro-active information from Prestashop to warn the merchant about the switch of mail configuration. Even if I undesrtand the reason and it's a good one, security is always a good one. I believe  Prestashop could have foreseen that deactivating the native mail() function from PHP and switching to  /usr/sbin/sendmail might not work for all configs of Prestashop. I have read lots of threads since the switch in  1.7.7.7 where merchant cannot use the mail function. That is a pity.
